### DISCUSSION TITLE
First launch - first problems

### DISCUSSION THREAD

**Discussion by max456im** [rupeshs/fastsdcpu#283] [Q&A]
![Снимок экрана 2024-11-29 160916](https://github.com/user-attachments/assets/8526fb29-f799-4b8f-ab48-472381f99c6c)
Hi, when installing on Windows, it gives an error when launching the web interface. Problem with tiny-vit-sam file names
1. How and where to install these files?
2. How to add model_index.json when loading lora models (because it asks)

**Comment by rupeshs** [comment]
You can  ignore those warnings
